charts/pulsar/templates/presto/presto-coordinator-configmap.yaml

```diff
@@ -35,16 +35,13 @@ data:
     -server
     -Xmx{{ .Values.presto.coordinator.jvm.memory }}
     -XX:+UseG1GC
-    -XX:+UnlockExperimentalVMOptions
-    -XX:+AggressiveOpts
-    -XX:+DoEscapeAnalysis
-    -XX:ParallelGCThreads=4
-    -XX:ConcGCThreads=4
-    -XX:G1NewSizePercent=50
-    -XX:+DisableExplicitGC
-    -XX:-ResizePLAB
+    -XX:G1HeapRegionSize=32M
+    -XX:+UseGCOverheadLimit
+    -XX:+ExplicitGCInvokesConcurrent
+    -XX:+HeapDumpOnOutOfMemoryError
     -XX:+ExitOnOutOfMemoryError
-    -XX:+PerfDisableSharedMem
+    -Dpresto-temporarily-allow-java8=true
+    -Djdk.attach.allowAttachSelf=true
```

Comment:
This really limit our ability to use updated Presto/Trino version, current version is 361 and this will limit us to version <= 331 which was release a year and a half ago: https://trino.io/docs/current/release/release-330.html